server/src/main/java/org/apache/iotdb/db/service/thrift/impl/TSServiceImpl.java

```diff
@@ -222,8 +224,10 @@ public TSExecuteStatementResp call() throws Exception {
       plan.setLoginUserName(username);
 
       QUERY_FREQUENCY_RECORDER.incrementAndGet();
-      AUDIT_LOGGER.debug("Session {} execute Query: {}", session, statement);
-
+      if (!AuditLogUtils.LOG_LEVEL_NONE.equals(conf.getAuditLogStorage())) {
```

Comment:
same as above, you check it all throught your pr and I won't point it out in the remaing reviews.